ci/taskcat.yaml

```diff
@@ -56,3 +56,14 @@ tests:
     regions:
       - us-east-1
     template_file: s3-bucket.yaml
+  s3web-bucket:
```

Comment:
This looks good.  I just tried to run it and realized that I forgot how long cloudfront takes to deploy.  These adds about 60 mins to the tests.  Not only that, the stack deletion fails when taskcat attempts to delete buckets.  Let's just comment out these for now with the following note:

For s3web-bucket:
"Stack deletion fails when taskcat attempts to delete bucket"

For s3web-cloudfront-bucket:
"Cloudfront adds to much time to the test (~60 mins to deploy and cleanup).  Also taskcat fails to cleanup."

Reply:
OK. Thanks for the feedback!